Sources/Components/Components/Watchlist/WKWatchlistViewModel.swift

```diff
@@ -1,27 +1,162 @@
 import Foundation
+import WKData
+import SwiftUI
 
-public final class WKWatchlistViewModel {
+public final class WKWatchlistViewModel: ObservableObject {
 
 	// MARK: - Nested Types
 
 	public struct LocalizedStrings {
 		public var title: String
 		public var filter: String
+		public var byte: String
+		public var bytes: String
 
-		public init(title: String, filter: String) {
+		public init(title: String, filter: String, byte: String, bytes: String) {
 			self.title = title
 			self.filter = filter
+			self.byte = byte
+			self.bytes = bytes
+		}
+	}
+
+	struct ItemViewModel: Identifiable {
+		public let id = UUID()
+
+		static var dateFormatter = {
```

Comment:
It may be helpful to move date formatters to a Date extension file in either Components or WKData - for quick date formatting overviews or reuse in other future view models. I don't feel strongly about it though, especially since these are pretty simple.

Reply:
Good callout – yeah I'm kinda on the fence about it (constructing feature specific formatters vs. global to framework ones) but maybe it leans towards best practice like `WKFont` and `WKColor` in that our fixed palette of formatters live in that extension. I went ahead and made the change.